build-system/app.js

```diff
@@ -1331,4 +1331,21 @@ function decryptDocumentKey(encryptedDocumentKey) {
   return parsedJson.key;
 }
 
+// serve local vendor config JSON files
+app.use('(/dist)?/rtv/*/v0/analytics-vendors/:vendor.json', (req, res) => {
+  const {vendor} = req.params;
+  const max = pc.env.SERVE_MODE === 'default' ? '.max' : '';
+  const localVendorConfigPath = `${pc.cwd()}/dist/v0/analytics-vendors/${vendor}${max}.json`;
+
+  fs.readFileAsync(localVendorConfigPath)
+    .then(file => {
+      res.setHeader('Content-Type', 'application/json');
+      res.end(file);
+    })
+    .error(() => {
+      res.status(404);
```

Comment:
Question, will we return 404 if a config file can't be found from cdn? Asking because we definitely want to match the behavior.

Reply:
Yes :)